frontend/src/metadata/popups/SliceFinderPopup.svelte

```diff
@@ -0,0 +1,284 @@
+<script lang="ts">
+	import Paper from "@smui/paper";
+	import { fade } from "svelte/transition";
+	import { Label } from "@smui/common";
+	import Chip from "@smui/chips";
+	import { Svg } from "@smui/common";
+	import { showNewSlice, slices, sliceToEdit } from "../../stores";
+	import IconButton, { Icon } from "@smui/icon-button";
+	import Select, { Option } from "@smui/select";
+	import { clickOutside } from "../../util/clickOutside";
+	import { mdiPlus, mdiClose } from "@mdi/js";
+	import { showSliceFinder } from "../../stores";
+	import { model, metric } from "../../stores";
+	import Button from "@smui/button";
+	import { getMetricsForSlices } from "../../api/slice";
+	import { ZenoService, type MetricKey } from "../../zenoservice";
+
+	let slice_data = [];
+	// dummy data for presenting frontend UI
```

Comment:
is this still dummy data? 

If not can you comment what these options all are?